excalidraw-app/components/AppMainMenu.tsx

```diff
@@ -29,15 +27,6 @@ export const AppMainMenu: React.FC<{
       <MainMenu.DefaultItems.ClearCanvas />
       <MainMenu.Separator />
       <MainMenu.DefaultItems.Socials />
-      <MainMenu.ItemLink
```

Comment:
I would suggest to add a flag here like below, that this is only rendered when `import.meta.env.VITE_APP_PLUS_LP` is available, otherwise it's not shown. I hope it lessens the merging burden in the future.

Reply:
Flag is added.